docs/superpowers/plans/2026-04-06-plan2-layout-redesign.md

```
@@ -0,0 +1,631 @@
+  () => import("react-leaflet").then(m => m.Polyline),
+  { ssr: false }
+);
+const Popup = dynamic(
+  () => import("react-leaflet").then(m => m.Popup),
+  { ssr: false }
+);
+```
+
+Layout inside fullscreen:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)